apps/docs/src/app/platform/component-docs/platform-dynamic-page/platform-dynamic-page-examples/platform-dynamic-page-page-overflow.service.ts

```diff
@@ -0,0 +1,7 @@
+import { Injectable } from '@angular/core';
+import { BehaviorSubject } from 'rxjs';
+
+@Injectable({ providedIn: 'root' })
+export class PlatformDynamicPagePageOverflowService {
+    isExampleOpened = new BehaviorSubject<boolean>(false);
+}
```

Comment:
Let’s register this service in the dynamic-page-doc module itself and don’t use `provideIn: root` (it’s needed only in this particular module).
I would prefer to keep this service with 2 methods as
```typescript
class PageContentService {
  disableOverflow() { content.styles.overflowY = 'hidden' }
  enableOverflow() { content.styles.overflowY = '' }
}
```
We can skip `fromEvent(window, 'popstate')` complexity and instead add `ngDestroy() { this.service.disableOverflow() }` for each example where it is needed.

@kavya-b  What do you think?